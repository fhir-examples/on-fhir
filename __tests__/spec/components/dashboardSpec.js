import React from 'react';

import { shallow } from 'enzyme';
import enzymeToJson from 'enzyme-to-json';

import App from '../../../app/components/dashboard';

describe('DashboardSpec', () => {

  it('should initialize Dashboard', () => {

    // given
    const wrapper = shallow(<App />);

    // then
    expect(enzymeToJson(wrapper)).toMatchSnapshot();
  });
});
