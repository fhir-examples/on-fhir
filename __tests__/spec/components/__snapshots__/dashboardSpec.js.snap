// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardSpec should initialize Dashboard 1`] = `
<ScrollView
  style={
    Array [
      Object {
        "flex": 1,
        "flexDirection": "column",
        "marginTop": 64,
      },
    ]
  }
>
  <DashboardItem
    label="Weight"
    onPress={[Function]}
  />
  <DashboardItem
    label="Height"
    onPress={[Function]}
  />
</ScrollView>
`;
